src/main/java/io/jenkins/plugins/coverage/CoverageProjectAction.java

```diff
@@ -53,7 +53,7 @@ public String getUrlName() {
      */
     public CoverageAction getLastResult() {
         for (Run<?, ?> b = run.getParent().getLastSuccessfulBuild(); b != null; b = BuildUtils.getPreviousNotFailedCompletedBuild(b)) {
-            if (b.getResult() == Result.FAILURE || (b.getResult() != Result.SUCCESS))
+            if (!((b.getResult() == Result.SUCCESS) || (b.getResult() == Result.UNSTABLE)))
```

Comment:
Maybe we can change it to `if (b.getResult() != Result.SUCCESS && b.getResult() != Result.UNSTABLE)` to make the logic more clear?